samples/javascript_nodejs/84.bot-authentication-certificate/index.js

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const path = require('path');
+// const fs = require('fs');
```

Comment:
it's used to read the file, I just made a comment to avoid lint issues
Check that I have two options to read the certificate values in the bot, with file or KeyVault, the file option is commented